server/plugins/DiskIO_UGens.cpp

```diff
@@ -126,17 +126,17 @@ void DiskIOMsg::Perform() {
             memset(buf->data + mPos * buf->channels, 0, mFrames * buf->channels * sizeof(float));
             goto leave;
         }
-        count = sf_readf_float(buf->sndfile, buf->data + mPos * buf->channels, mFrames);
+        count = sf_readf_float(GetSndFile(buf), buf->data + mPos * buf->channels, mFrames);
```

Comment:
this may not matter, but I wonder if it is cleaner to store the result of GetSndFile(but) into a variable and pass that around, rather than calling GetSndFile over and over again?

Reply:
IMO it's fine because `GetSndFile` is just a getter macro and not a real function call. I find it quite readable. After all, it's just 3 characters longer than the original code.

Reply:
I've changed it to `GETSNDFILE`